packages/coin/coin.go

```diff
@@ -131,6 +131,10 @@ func TypeFromBytes(b []byte) (Type, error) {
 	return r, err
 }
 
+func (t Type) IotagoRpcCointype() iotajsonrpc.CoinType {
```

Comment:
```suggestion
func (t Type) IotagoRPCCoinType() iotajsonrpc.CoinType {
```

Also I would remove `CoinType` from the name to avoid stuttering.

```suggestion
func (t Type) ToIotaJSONRPC() iotajsonrpc.CoinType {
```